### DISCUSSION TITLE
How to make a test transaction?

### DISCUSSION THREAD

**Discussion by 56463456354** [etsy/open-api#1192] [Q&A]
Hi,

We are developing a PHP integration with Etsy API. We finished the first phase to create the listings, now we need to import orders and send them to fulfillment. How can we generate a test order so we have some data to work with?

The sandbox instructions I found are outdated, and there is no "other" option in the payment methods. I thought about creating another account to buy a product from my shop, but I don't want to risk getting flagged for suspicious activity or shilling.

Is there any way to make a test order?

**Comment by Gareth064** [comment]
There is no way to do it other than to make an order from your shop.
